bundles/http/Citrix/Slack/metadata.json

```diff
@@ -1,45 +1,45 @@
 {
   "type" : "HTTP",
   "vendor" : "Citrix",
-  "id" : "babc7496-881f-47dd-9255-22a09e16da7d",
+  "id" : "c4539a17-ac0b-4066-9874-e45c69a2519c",
   "trackingUuid" : "f38fb99d-f4c9-4ff3-b408-2dd53f6d9974",
-  "title" : "Slack",
+  "title" : "slack_bugFix",
   "description" : "Deploy the Slack integration to provide additional monitoring capabilities for critical channels that may not be traffic intensive but require the attention of its members. In order to tailor the channels available to a specific group or department, use multiple integrations.",
   "iconUrl" : "https://iws-prod-global-customicons-cdn-endpoint.azureedge.net/image/cS1lBLq1CmDwphtXxg2u0tvMqFw67pZZ.png",
-  "masVersion" : "1.87.0.e549b8d5d8fe032d57acca7ee39842e2e208d74a",
-  "categories" : [ "WEB_SERVICES" ],
-  "created" : "2021-05-19T20:50:40",
+  "masVersion" : "1.104.0.e461ba996b9abda5fecc209c99d50aa617459879",
+  "categories" : [ ],
+  "created" : "2021-07-19T14:38:45",
   "supportsOAuthForActions" : true,
-  "i18nLanguages" : [ "de", "en", "es", "fr", "it", "ja", "nl", "pt-BR", "zh-CN" ],
+  "i18nLanguages" : [ "en" ],
```

Comment:
Translation variations are missing, therefore the build is failing, because it sees the translation files in the bundle. Please make your fix on the version imported from catalog into your environment.